index.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
+
+function now() {
+	return Date.now()/1000 | 0;
```

Comment:
The server sends back a delta in seconds (max age "10 seconds from now") and we store `now() + whatTheySent` into `global.D2LAccessTokenExpiresAt`, and later check `now() < global.D2LAccessTokenExpiresAt`... so I think we're safe as long as they don't actively change their clock... I'm not sure if we there is a way we can do better than that